firmware/Core/Src/littlefs/littlefs_benchmark.c

```diff
@@ -14,7 +14,18 @@
 /// @param response_str_len 
 /// @return 0 on success. >0 if there was an error.
 uint8_t LFS_benchmark_write_read(uint16_t write_chunk_size, uint16_t write_chunk_count, char* response_str, uint16_t response_str_len) {
-    const char file_name[] = "benchmark_test.txt";
+    const char dir_name[] = "benchmark_write_read";
+    // TODO: check if dir exists first; currently it gives a warning
+    LFS_make_directory(dir_name);
+    
+    char file_name[100];
+    snprintf(
+        file_name,
+        sizeof(file_name),
+        "%s/benchmark_test_%lu.txt",
+        dir_name,
+        HAL_GetTick()
+    );
```

Comment:
This was made for testing purposes, however wanted to confirm if this is okay to remove.

Reply:
Please keep benchmark telecommand, if that's what this comment was about

Reply:
My comment was more towards the additional code that creates a new file every time benchmark telecommand is called, but with new recent changes that benchmark writes to the same file and to a new file this comment can be safely ignored